app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -249,7 +249,7 @@ private String getChapterIdFromIntent(final Intent intent) {
         return null;
     }
 
-    private void fetchChapters() {
+    void fetchChapters() {
```

Comment:
It's because when private methods, fields are accessed by anonymous classes, compiler generates unnecessary methods. This avoids that.